examples/siro_sandbox/hitl_arg_parser.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+from typing import Final
+
+# Please reach out to the paper authors to obtain this file
+DEFAULT_POSE_PATH: Final[str] = (
+    # TODO: Get from model.
+    "data/humanoids/humanoid_data/walking_motion_processed_smplx.pkl"
+)
+
+DEFAULT_CFG: Final[
+    str
+] = "experiments_hab3/pop_play_kinematic_oracle_humanoid_spot.yaml"
+
+
+def create_hitl_arg_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Note *all* of these command-line parameters will be converted to Hydra parameters in an upcoming PR.  (Actually, a few of them can probably be removed altogether.)